src/SelfServe/SqlX/SqlX.tsx

```diff
@@ -206,9 +180,22 @@ export default class SqlX extends SelfServeBaseClass {
           },
         };
       } else {
-        // Check for scaling up/down/in/out
+        let operationStatusUrl;
+        if (baselineValues.get("instances")?.value === currentValues.get("instances")?.value) {
+          currentValues.set("warningBanner", {
```

Comment:
I dont think it will ever get to a state where instances value is the same, yet saving is possible right? The save button will be disabled in this case. Any particular reason you added this?

Reply:
This will happen if you move from 3 to 2 and move back to 3. Save button will unfreeze when you move from 3 to 2 and stay unfrozen when you move back from 2 to 3.

Reply:
What is the scenario: 3 -2 is pending? and when that is the user changes from 2 -3? 

in those cases, how does the rp API behave? will it fail that an update is in progress / does it let it go through?

Reply:
there was a bug on the selfserve model that did not disable save/discard button when an initial value of a property is set to "undefined". PR is out to fix that : https://github.com/Azure/cosmos-explorer/pull/764

This check will no longer be needed after that fix goes in.